### PR TITLE
[Particles] Fix invalid parameter ranges

### LINKED ISSUE #83995: Incorrect value set in ParticleProcessMaterial.
### Godot version

4.2.beta2.official

### System information

Windows 11

### Issue description

When creating the ParticleProcessMaterial, incorrect values are used (linear_accel_max and radial_accel_max).

Current:
grafik

Fix:
grafik

### Steps to reproduce

Just create a ParticleProcessMaterial and convert it to Shader.

### Minimal reproduction project

N/A

### PR DESCRIPTION
**Pull request by AThousandShips**
Searched and couldn't find any other cases like this

* Fixes: #83995

### CODE REVIEW

File: scene/resources/particle_process_material.cpp [hunk 1]
```diff
@@ -531,8 +531,8 @@ void ParticleProcessMaterial::_update_shader() {
 
 	code += "\n";
 	code += "void calculate_initial_physical_params(inout PhysicalParameters params, inout uint alt_seed){\n";
-	code += "	params.linear_accel = mix(linear_accel_min, linear_accel_min ,rand_from_seed(alt_seed));\n";
-	code += "	params.radial_accel = mix(radial_accel_min, radial_accel_min,rand_from_seed(alt_seed));\n";
+	code += "	params.linear_accel = mix(linear_accel_min, linear_accel_max, rand_from_seed(alt_seed));\n";
+	code += "	params.radial_accel = mix(radial_accel_min, radial_accel_max, rand_from_seed(alt_seed));\n";
 	code += "	params.tangent_accel = mix(tangent_accel_min, tangent_accel_max,rand_from_seed(alt_seed));\n";
 	code += "	params.damping = mix(damping_min, damping_max,rand_from_seed(alt_seed));\n";
```
Comment (akien-mga) [inline scene/resources/particle_process_material.cpp:537]:
```suggestion
	code += "	params.tangent_accel = mix(tangent_accel_min, tangent_accel_max, rand_from_seed(alt_seed));\n";
	code += "	params.damping = mix(damping_min, damping_max, rand_from_seed(alt_seed));\n";
```

Comment (AThousandShips) [reply scene/resources/particle_process_material.cpp:537]: Left them alone to keep the scope small but will add them too

Comment (akien-mga) [reply scene/resources/particle_process_material.cpp:537]: Though there's a bunch more style issues like this in the surrounding code, might be out of scope.

**Review by akien-mga — APPROVED** [review]

**Review by QbieShay — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by AThousandShips** [comment]
Thank you!

**Pull request merged**